src/api/apiConnector.ts

```diff
@@ -23,4 +32,29 @@ export class ApiConnector {
     public connect(): Promise<void> {
         return this._connectionPromise;
     }
+
+    private handleConnectionState(state: string, callback: (ConnectionState) => void): void {
```

Comment:
all private methods are suppose to start with underscore

Reply:
Thats just one of the naming conventions. One I used using underscore for private property, not methods.
class, enum, interface: PascalCase
property, variable: camelCase
private property: _camelCase
method, function:  camelCase